repair_shards.c

```diff
@@ -209,3 +204,67 @@ RecreateTableDDLCommandList(Oid relationId, int64 shardId)
 
 	return extendedRecreateCommandList;
 }
+
+
+/*
+ * CopyDataFromFinalizedPlacement connects to an unhealthy placement and
+ * directs it to copy the specified shard from a certain healthy placement.
+ * This function assumes that the unhealthy placement already has a schema
+ * in place to receive rows from the healthy placement. This function returns
+ * a boolean indicating success or failure.
+ */
+static bool
+CopyDataFromFinalizedPlacement(ShardPlacement *placementToRepair,
```

Comment:
Two quick notes on the function declaration.

1. Do we have a discrepancy between the order of ``ShardPlacement`` arguments? The caller first uses the source placement (healthy placement) and the target placement (placementToRepair). I think we declare those arguments in the inverse order in here, but I'm not sure.
2. To be consistent in the order of arguments between ``RecreateTableDDLCommandList`` and this function, what do you think of the following argument ordering? ``Oid distributedTableId, int64 shardId, ShardPlacement *sourcePlacement, ShardPlacement *targetPlacement``? (This way, we'd follow the consistency advice in CC2 7.5)